src/config/constants.js

```diff
@@ -31,8 +32,38 @@ export const PERIOD_TYPE = Object.freeze({
 });
 
 export const POST_LISTS = Object.freeze({
+  ORDER_KR: {
+    NEWEST: "최신 순",
+    LIKE: "공감 많은 순",
+    COMMENT: "댓글 많은 순",
+  },
+  ORDER_EN: {
+    NEWEST: "NEWEST",
+    LIKE: "LIKE",
+    COMMENT: "COMMENT",
+  },
+  INCLUDED_KEYWORD: "포함할 키워드",
+  EXCLUDED_KEYWORD: "제외할 키워드",
+  ISAD_KR: {
+    ALL: "광고 포함",
+    ONLY_ADS: "광고만",
+    NO_ADS: "광고 제외",
+  },
+  ISAD_EN: {
```

Comment:
```suggestion
  IS_AD_EN: {
```
IS와 AD 사이에도 `_`를 추가해주세요!